quickwit/Cargo.lock

```diff
@@ -8954,6 +8964,15 @@ dependencies = [
  "zstd-safe 6.0.6",
 ]
 
+[[package]]
+name = "zstd"
```

Comment:
can you update zstd and lru in quickwit? This PR doubles the dep.

Reply:
updated lru, but zstd is a nested dependency